scripts/src/admin.ts

```diff
@@ -0,0 +1,35 @@
+/** Load the Firebase app and expose it to scripts */
```

Comment:
maybe note in the comment the assumption about where this is being run. (I guess it assumes it's being executed from the scripts directory?

Reply:
clarified in 6c546282352ff23843530efb9480eddaff2accda